 win32/Makefile.msc | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/win32/Makefile.msc b/win32/Makefile.msc
index 9c65153..8d566a3 100644
--- a/win32/Makefile.msc
+++ b/win32/Makefile.msc
@@ -13,8 +13,8 @@ TOP = .
 LOC =
 
 # variables
-STATICLIB = zlib.lib
-SHAREDLIB = zlib1.dll
+STATICLIB = zlib_a.lib
+SHAREDLIB = zlib.dll
 IMPLIB    = zdll.lib
 
 CC = cl
@@ -22,7 +22,7 @@ AS = ml
 LD = link
 AR = lib
 RC = rc
-CFLAGS  = -nologo -MD -W3 -O2 -Oy- -Zi -Fd"zlib" $(LOC)
+CFLAGS  = -nologo -MD -W3 -O2 -Oy- -Zi -Fd"zlib_a" $(LOC)
 WFLAGS  = -D_CRT_SECURE_NO_DEPRECATE -D_CRT_NONSTDC_NO_DEPRECATE
 ASFLAGS = -coff -Zi $(LOC)
 LDFLAGS = -nologo -debug -incremental:no -opt:ref
